### ISSUE TITLE
Analogue Pocket port

### ISSUE THREAD

**Issue by CaitSith2** [RetroSven/ult3gbc#2]
I have taken the liberty to port this to [Analogue Pocket](https://www.analogue.co/pocket).  

[Source Code](https://github.com/CaitSith2/ult3gbc) at commit 265ae4
ult3.pocket [Binary](https://github.com/CaitSith2/ult3gbc/raw/main/ult3.pocket)

At the low level, LCDC lives at 4E instead of 40,  and both LCDC and STAT has their bit defintions flipped.  Additionally, there is a custom logo that would render as the analogue logo instead of the nintendo logo.

**Comment by RetroSven** [comment]
Nice - thanks much :)

**Comment by CaitSith2** [comment]
Did try to post a comment on [https://ultima3.ultimacodex.com/ultima-3-for-game-boy-color/](https://ultima3.ultimacodex.com/ultima-3-for-game-boy-color/), but that didn't seem to go through.

**Comment by RetroSven** [comment]
Yeah I can't post comments there either and I'd never know anything was posted there - so I'm glad that one of the 7 comments is to direct people here!  I started this project 20+ years ago for a game that was (at the time) 20+ years old - so it's nice that there is occasional interest in it :P

**Comment by CaitSith2** [comment]
Turns out it was simply moderated.  The comment did make it through on that page now.

**Comment by CaitSith2** [comment]
PR #4 now deals with being able to build for both Gameboy color and Analogue Pocket with the same set of source files.
